modules/kubeflow-feast/variables.tf

```diff
@@ -1,3 +1,14 @@
+variable "risk" {
+  type        = string
+  description = "Value for the risk to be used"
+  default     = "stable"
+
+  validation {
+    condition     = contains(["stable", "candidate", "beta", "edge"], var.risk)
```

Comment:
just realized our charms don't have a `<track>/beta` channel for the 1.10 versions, we only have `1.10/beta` channel for the bundle which deploys `latest/beta` charms.